rnd/autogpt_builder/src/lib/autogpt-server-api/client.ts

```diff
@@ -6,17 +7,17 @@ import {
   GraphMeta,
   GraphExecuteResponse,
   NodeExecutionResult,
-} from "./types";
+} from "./types"
 
 export default class AutoGPTServerAPI {
   private baseUrl: string;
   private wsUrl: string;
   private socket: WebSocket | null = null;
   private messageHandlers: { [key: string]: (data: any) => void } = {};
+  private supabaseClient = createClient();
 
   constructor(
-    baseUrl: string = process.env.NEXT_PUBLIC_AGPT_SERVER_URL ||
-      "http://localhost:8000/api",
+    baseUrl: string = process.env.AGPT_SERVER_URL || "http://localhost:8000/api"
```

Comment:
If you want to go back to not using NEXT_PUBLIC prefixed env vars, can you change it everywhere

Reply:
i put it back this was not intended!